test/unit/config.test.js

```diff
@@ -410,6 +145,7 @@ describe('config', function () {
       );
     });
 
+    // TODO: is this part of the ecosystem and do we want it?
```

Comment:
@vdiez `process.env.npm_config_sonar_scanner_version` we haven't supported this env naming convention yet, is it important for the ecosystem?

Reply:
I'm not sure here; I would check with @henryju. This is one of these things that have been long time before we took the project. I'm not against dropping it for the sake of standardizing with the rest of the scanners